packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -7,17 +7,64 @@ const { promisify } = require('util');
 const {
   recursivelyDeleteS3Bucket, promiseS3Upload, getS3Object, s3GetObjectTagging
 } = require('@cumulus/aws-client/S3');
-const { s3 } = require('@cumulus/aws-client/services');
+const { s3, secretsManager } = require('@cumulus/aws-client/services');
 const BucketsConfig = require('@cumulus/common/BucketsConfig');
-const { readJsonFixture } = require('@cumulus/common/test-utils');
+const { randomId, readJsonFixture } = require('@cumulus/common/test-utils');
+const launchpad = require('@cumulus/launchpad-auth');
 const { xmlParseOptions } = require('../../utils');
 
+const { getCmrSettings } = require('../../cmr-utils');
 const cmrUtil = rewire('../../cmr-utils');
 const { isCMRFile, getGranuleTemporalInfo } = cmrUtil;
 const uploadEcho10CMRFile = cmrUtil.__get__('uploadEcho10CMRFile');
 const uploadUMMGJSONCMRFile = cmrUtil.__get__('uploadUMMGJSONCMRFile');
 const generateFileUrl = cmrUtil.__get__('generateFileUrl');
 
+const launchpadSecret = randomId('launchpad-secret');
+const launchpadPassphrase = randomId('launchpad-passphrase');
+const cmrPasswordSecret = randomId('cmr-password-secret');
+const cmrPassword = randomId('cmr-password');
+
+test.before(async (t) => {
+  process.env.cmr_provider = 'CUMULUS-TEST';
+  process.env.cmr_client_id = 'Cumulus-Client-Id';
+  process.env.cmr_oauth_provider = 'earthdata';
+
+  process.env.launchpad_passphrase_secret_name = launchpadSecret;
+  process.env.launchpad_api = 'launchpad-api';
+  process.env.launchpad_certificate = 'launchpad-cert';
+
+  await secretsManager().createSecret({
+    Name: launchpadSecret,
+    SecretString: launchpadPassphrase
+  }).promise();
+
+  process.env.cmr_password_secret_name = cmrPasswordSecret;
+  process.env.cmr_username = 'cmr-user';
+
+  await secretsManager().createSecret({
+    Name: cmrPasswordSecret,
+    SecretString: cmrPassword
+  }).promise();
+
+  t.context.launchpadStub = sinon.stub(launchpad, 'getLaunchpadToken')
+    .callsFake((config) => Promise.resolve(`${config.passphrase}-${config.api}-${config.certificate}`));
```

Comment:
Unimportant, but if you just want a stub that returns a resolved promise value you can use `sinon.stub().resolves(value)`

Reply:
So the value is dynamic based on what's passed in. Could that work with just `resolves`? I did not make any updates here.

Reply:
Ah, no, you're right. You would want to leave this one as is